src/utils/authHelpers.jsx

```diff
@@ -0,0 +1,31 @@
+import firebase from "../firebaseConfig";
+
+export const EMAIL_NOT_FOUND_ERROR = "auth.emailNotFound";
+export const EMAIL_ALREADY_TAKEN_ERROR = "auth.emialAlreadyTaken";
```

Comment:
theres a typo here in email, which exactly shows why we dont use magic strings

Reply:
That's a practical lesson :D